AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -52,6 +52,7 @@ genius.com##apple-music-player
 ltn.com.tw#%#AG_onLoad(function() { $('div[class^="photo picbox 1 photoOn"] > a[href="//draw.ltn.com.tw/slot_v8/"]').closest('div.picbox.\\31').remove(); });
 ! https://github.com/AdguardTeam/AdguardFilters/issues/28876
 instagram.com##.ZUqME.N9d2H[style="width: 100%;"]
+tutsgalaxy.com###secondary > .widget_blog_subscription
```

Comment:
Please move the rule internally as this way it suggests it is related to id 28876